.circleci/config.yml

```diff
@@ -31,15 +31,37 @@ jobs:
       - run: composer require silverstripe/recipe-cms
       - run: composer update -n --prefer-dist --ignore-platform-reqs
       - run: composer vendor-expose
+      - run:
+          name: Setup Integration Pre-requisites
+          command: |
+            if [ ! -d assets ]; then mkdir assets; fi
```

Comment:
Interesting, this is needed?

Reply:
This is optional as `dev/build` creates the folder. 
There are other environments that have issues with permission.
Will remove it 👍